src/app/routes/routes.jsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
-import { Route, Redirect } from 'react-router';
+import { Route, IndexRoute, Redirect } from 'react-router';
 
 import Application from '../components/Application/Application';
+import SearchContainer from '../components/SearchContainer/SearchContainer';
 import NotFound404 from '../components/Error/NotFound404';
+import WorkDetail from '../components/WorkDetail/WorkDetail';
+import Home from '../components/Home/Home';
 
 const routes = {
   default: (
     <Route path="/" component={Application}>
-      <Route path="/search" component={Application} />
-      <Route path="/work" component={Application} />
+      <IndexRoute component={Home} />
```

Comment:
This doesn't work if it's `SearchContainer` instead of `Home`?

Reply:
I'll try with `SearchContainer` to see if that satisfies React Router.